azure-pipelines.yml

```diff
@@ -1,36 +1,108 @@
-trigger:
-    - main
+pr:
+    branches:
+        include:
+            - main
```

Comment:
Could we add back the trigger so that this pipeline runs on **both** PRs and merges to `main`?

Reply:
Oh wait never mind!

Reply:
I thought `ci.yml` (GH actions) was being run on both PR and merge but upon a second look I guess it's not, so I'm actually happy to leave this as is.

Reply:
Yeah, I deliberately mirrored exactly how you had it working in the GitHub Action YAML file.

Reply:
Perfect, thank you 🙂